packages/fluentui/react-northstar/src/themes/teams/components/Menu/menuItemStyles.ts

```diff
@@ -201,6 +203,14 @@ export const menuItemStyles: ComponentSlotStylesPrepared<MenuItemStylesProps, Me
 
         ...(underlined && { color: v.underlinedColorHover }),
 
+        ...(!disabled &&
+          !primary &&
+          vertical && {
+            [`&>.${menuItemIconClassName}`]: {
+              color: v.subMenuIconColor,
+              ...getIconFillOrOutlineStyles({ outline: false }),
+            },
+          }),
```

Comment:
It would be better to this to `packages/fluentui/react-northstar/src/themes/teams/components/Menu/menuItemIconStyles.ts`. **But** we will need to pass more non-meaningful props to `MenuItemIcon` only for that, so we can live this this. No changes needed.